src/services/EdFi.Dms.Api.Tests/EdFi.Dms.Api.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Autofac" Version="8.0.0" />
+    <PackageReference Include="Autofac.Extras.FakeItEasy" Version="7.0.0" />
+    <PackageReference Include="FakeItEasy" Version="8.1.0" />
+    <PackageReference Include="FluentAssertions" Version="6.12.0" />
```

Comment:
The ODS/API code, and maybe Admin API, uses the Shouldly library instead of FluentAssertions. We can consider switching back. I always liked FluentAssertions a little better (better documentation, better method calls), but in testing I performed in 2018, the library had some noticeable performance impact relative to others.

I've rerun those tests on current versions and it is still true. I'm a little less worried about it, though I'm continuing to experiment at home (call it an after-hours hobby 😉).